ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -189,6 +189,10 @@ objects:
         value: ${IMAGE_FORMAT}
       - name: KUBECONFIG
         value: /tmp/artifacts/installer/auth/kubeconfig
+      - name: OS_CLOUD
+        value: 'openstack-cloud'
+      - name: OS_CLIENT_CONFIG_FILE
+        value: '/tmp/cluster-profile/clouds.yaml'
```

Comment:
I think this is a good start. It would be even better to expose them all the way up to the template paramter level, but lets start here. we don't want to add too many variables to the parameters section of the template.